changelog.md

```diff
@@ -0,0 +1,47 @@
+# Changelog
+
+## Version 0.1
+### Added
+- Added scripts to facilitate launching the WebGoat application.
+- Added .NET 5 SDK as a minimum requirement to run WebGoat locally.
+- Added UI enhancements to improve the look and feel of the application.
+- Added examples of a valid credit card number.
+- Added a notification about an empty credit card field.
```

Comment:
I would rephrase that, because we have done a lot more, e.g.:
- Improved validation of credit card numbers provided by user